### PR TITLE
Culture code, org and wow first release to public

### PR DESCRIPTION
**Pull request by remiks**

### CODE REVIEW

File: AGILE-WOW.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+* Agile allows Fortytwo teams to work in a flexible and self-managing way. We have increased autonomy and authority over our decisions through frequent and open collaboration. 
+
+* The Agile framework also produces important metrics like lead time, cycle time and throughput that helps teams measure their own performance, identify bottlenecks, overcome pitfalls like scope creep, and gives them the ability to make data-driven decisions to correct them.
+
+---
+
+## Basic Fortytwo Agile Guidelines
+
+We don't have a one specific or hard-set Agile way of working at Fortytwo. What we have is a common understanding of and respect for our inherent differences, which means we all work and interact differently -- and that's more than ALRIGHT!
```
Comment (roberthstrand) [inline AGILE-WOW.md]: Alright is usually used as an expression, which might stem from _all right_, but I would rather have the full set of words in a text like this.

File: ORGANIZATION.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### What we expect from all engineers
+
+All engineers are expected to know, and/or make sure to train for the following knowledge as a minimum:
+
+* Basic Git knowledge.
+* Basic Azure DevOps understanding and usage.
+* Basic Markdown syntax and understanding for documentation.
+* Basic SCRUM knowledge.
+* Basic knowledge with Powershell or C#
```
Comment (roberthstrand) [inline ORGANIZATION.md]: Could we change this to something less specific? Like, `Basic knowledge of scripting and programming languages`?

File: ORGANIZATION.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Our common expectations 
+To ensure clear communication and alignment, here is a description of the expectations for each role within the company.
+
+### What we expect from all engineers
+
+All engineers are expected to know, and/or make sure to train for the following knowledge as a minimum:
+
+* Basic Git knowledge.
+* Basic Azure DevOps understanding and usage.
```
Comment (roberthstrand) [inline ORGANIZATION.md:16]: Maybe add GitHub too?

**Comment by remiks** [comment]
Updated!

**Review by roberthstrand — APPROVED** [review]

**Pull request merged**